packages/native/test/lib/ElementAssertion.test.tsx

```diff
@@ -0,0 +1,94 @@
+import { AssertionError, expect } from "@assertive-ts/core";
+import { render } from "@testing-library/react-native";
+import {
+  View,
+  TextInput,
+} from "react-native";
+
+import { ElementAssertion } from "../../src/lib/ElementAssertion";
+
+describe("[Unit] ElementAssertion.test.ts", () => {
+  describe(".toBeDisabled", () => {
+    context("when the element is TextInput", () => {
+      it("returns the assertion instance when is not editable", () => {
+        const element = render(
+            <TextInput testID="id" editable={false} />,
+        );
+        const test = new ElementAssertion(element.getByTestId("id"));
+        expect(test.toBeDisabled()).toBe(test);
+      });
+      it("throws an error when it is editable", () => {
```

Comment:
```suggestion
      });

      it("throws an error when it is editable", () => {
```

Let's check this for the whole file 🙂